src/view/hooks/useUploadImage.ts

```diff
@@ -0,0 +1,76 @@
+import { getApp } from "firebase/app";
+import {
+    getDownloadURL,
+    getStorage,
+    ref,
+    uploadBytesResumable,
+    UploadTask,
+} from "firebase/storage";
+import { useState } from "react";
+
+const useUploadImage = () => {
+    const [file, setFile] = useState<File | null>(null);
+    const [imageURL, setImageURL] = useState<string>("");
+    const [uploadProgress, setUploadProgress] = useState<number | null>(null);
+    const [isUpload, setIsUpload] = useState<boolean>(false);
+
+    const handleFileChange = (selectedFile: File) => {
+        setFile(selectedFile);
+    };
+
+    const handleUpload = async () => {
+        try {
+            const firebaseApp = getApp();
+            const storage = getStorage(firebaseApp, "gs://poroto-place-images");
```

Comment:
[refactor] ここ、環境変数に切り分けられると良さそうです。
```suggestion
            const storage = getStorage(firebaseApp, process.env.CLOUD_STORAGE_POROTO_PLACE_IMAGES);
```

修正箇所：
https://github.com/poroto-app/poroto/blob/e223494a24cf1d031e9cef49b579aecb289d265c/next.config.mjs#L54

https://github.com/poroto-app/poroto/blob/develop/.env.development